DEPENDENCIES.rst

```diff
@@ -57,13 +57,13 @@ Conda
 ~~~~~~~~~~~~~~~~~~~~~
 
 
-1. If you don't have Conda, start by installing the Python 3.x version of
+#. If you don't have Conda, start by installing the Python 3.x version of
    Anaconda_ (or miniconda_ for a more lightweight choice) to prepare it for
    Cyclus.
 
 .. website_include_conda_end
 
-2. Once you have Conda installed, installing Cyclus straightforward.
+#. Once you have Conda installed, installing Cyclus straightforward.
```

Comment:
When this renders on GitHub, this line and l.60 both show up with a 1 for the numbering. This may be because of the `.. website_include_conda_end` in the middle. If you build the website, does the numbering all look good? If so, then I think that's a bigger priority and should keep it as is.

Reply:
Thanks for the reminder.  I've modified it now to include all the information needed for the website, and should be considered in conjunction with cyclus.github.com PR [#364](https://github.com/cyclus/cyclus.github.com/pull/364). The numbering is looking good on my local build now.